internal/hud/view/view.go

```diff
@@ -9,22 +9,55 @@ import (
 
 type ResourceInfoView interface {
```

Comment:
hmm should the resourceinfo stuff be its own file?

Reply:
maybe, but that's easy to do later, and we're about to have a few commits moving fields between these types, so might as well leave them together for now?